TelegramBotAPI-core/src/commonMain/kotlin/com/github/insanusmokrassar/TelegramBotAPI/types/message/CommonSupergroupEventMessage.kt

```diff
@@ -1,14 +1,18 @@
 package com.github.insanusmokrassar.TelegramBotAPI.types.message
 
 import com.github.insanusmokrassar.TelegramBotAPI.types.MessageIdentifier
+import com.github.insanusmokrassar.TelegramBotAPI.types.User
 import com.github.insanusmokrassar.TelegramBotAPI.types.chat.abstracts.SupergroupChat
 import com.github.insanusmokrassar.TelegramBotAPI.types.message.ChatEvents.abstracts.SupergroupEvent
 import com.github.insanusmokrassar.TelegramBotAPI.types.message.abstracts.ChatEventMessage
+import com.github.insanusmokrassar.TelegramBotAPI.types.message.abstracts.FromUserMessage
+import com.github.insanusmokrassar.TelegramBotAPI.types.message.abstracts.GroupEventMessage
 import com.soywiz.klock.DateTime
 
-data class SupergroupEventMessage(
+data class CommonSupergroupEventMessage(
     override val messageId: MessageIdentifier,
+    override val user: User,
     override val chat: SupergroupChat,
     override val chatEvent: SupergroupEvent,
     override val date: DateTime
-) : ChatEventMessage
+) : GroupEventMessage
```

Comment:
Here better to use `SupergroupEventMessage`. It must extends `GroupEventMessage`

Reply:
Done